src/System.Runtime/tests/Performance/Perf.Int32.cs

```diff
@@ -13,7 +13,9 @@ public class Perf_Int32
         private const int InnerCount = 500_000;
 
         private static string s_resultString;
+#if netcoreapp
```

Comment:
Should these get pulled out into partial classes, like we do for the rest of the `netcoreapp` specific code?

Reply:
the perf tests already got migrated to BenchmarkDotNet, this is mearly an intermediate work till we fully switch to it. I think conditional compilation is totally fine here.